### DISCUSSION TITLE
Support for ultra-wide aspect ratio, like 21:9?

### DISCUSSION THREAD

**Discussion by imclaux** [SebLague/Geographical-Adventures#124] [Q&A]
Hello, 

I just discovered this project and I really liked played a few minutes of it. I was wondering if there is some lines in the code that could be changed as in to let people with a 21:9 monitor enjoy the game in full-screen, my resolution is 3440x1440 and I would love to fill the screen. 
Hope it's not too late to ask that.

Thank you!

**Comment by L4DK** [comment]
Hey @imclaux,
You can make your own ratios with the + button:
![upload_2017-9-6_20-41-20](https://github.com/SebLague/Geographical-Adventures/assets/115590310/d5952509-015d-4662-b6b3-8314324d22eb)

You can also mask off / crop with the Main Camera, in the Viewport Rect area:
![upload_2017-9-6_20-44-34](https://github.com/SebLague/Geographical-Adventures/assets/115590310/d240513a-df69-46eb-8750-079000105982)

If you want to tune it for your wide display, you'll need to adjust the camera FOV to get the width you want.
